bears/xml/XMLBear.py

```diff
@@ -15,8 +36,19 @@ def process_output(self, output, filename, file):
         if self.stderr_output:  # pragma: no cover
             return self._process_issues(self.stderr_output, filename)
 
-    def run(self, filename, file):
+    def run(self, filename, file,
+            schema: path="",
+            dtdvalid: path_or_url(path, str)=""):
         '''
-        Checks the code with `xmllint`.
+        Checks the code with ``xmllint``.
+
+        :param: schema:        Use a W3C XML Schema file named ``schema`` for
```

Comment:
```
Use a W3C XML Schema file named ``schema``
```
Why is this named schema ? I didnt quite understand the description.
Isn't `The W3C XML Schema file to use for validation` sufficient ??